hq_superset/hq_domain.py

```diff
@@ -34,8 +34,11 @@ def after_request_hook(response):
 
 
 def is_user_admin():
-    from superset import security_manager
-    return security_manager.is_admin()
+    try:
+        from superset import security_manager
+        return security_manager.is_admin()
+    except AttributeError:
+        return False
```

Comment:
It bothers me that we have to catch this error, because it's not in our codebase.

What if we first check the user, like so?

```suggestion
    from superset import security_manager
    from flask import g
    return g.user and security_manager.is_admin()
```

Reply:
@kaapstorm 
> It bothers me that we have to catch this error, because it's not in our codebase.

Yeah.

The `g.user` is the part where it fails hard. Just running `g.user` will raise the AttributeError.

Reply:
Does security manager provide another method on it that we can call to check if there is a user or not before calling `is_admin`?

Reply:
Can we instead at least do this
```
try:
    # Check if there is a user first before checking for admin via security manager
    g.user
except AttributeError:
    return False
return security_manager.is_admin()
```